net/rs/client-gen-dotnet/tests/snapshots/generator__basic_works.snap

```diff
@@ -0,0 +1,63 @@
+---
+source: rs/client-gen-dotnet/tests/generator.rs
+expression: "gen(idl, \"Basic\")"
+---
+using global::Sails.Remoting;
+using global::Sails.Remoting.Abstractions;
+using global::Sails.Remoting.Abstractions.Core;
+using global::System;
+using global::System.Collections.Generic;
+
+#pragma warning disable RCS0056 // A line is too long
+
+namespace Basic.Client;
+
+public interface IBasic
+ { ICall<global::Substrate.NetApi.Model.Types.Primitive.U16> DoThis(global::Substrate.NetApi.Model.Types.Primitive.U32 p1, MyParam p2);
+ICall<global::Substrate.NetApi.Model.Types.Primitive.U8> DoThat(global::Substrate.NetApi.Model.Types.Base.BaseTuple<global::Substrate.NetApi.Model.Types.Primitive.U8, global::Substrate.NetApi.Model.Types.Primitive.U32> p1);
+ }
+
+ public partial class Basic : IBasic
```

Comment:
Why is partial? Shouldn't it be sealed?

Reply:
it's good practice to make generated classes partial

Reply:
+ sealed